### PR TITLE
chore: split cron jobs and add contributors update on action

### PR DESCRIPTION
**Pull request by mainframev**
- It's better to split the cron jobs, so they are independent, and merge requests will be more descriptive
- add an update of contributors on the action, so it does not run with every building process

for now, used the `.mjs` extensions, later we can rewrite that to `ts-node` and `.ts`
 Storybook: https://orbit-mainframev-chore-split-cron-jobs.surge.sh

### CODE REVIEW

File: .github/workflows/cron-update-contributors.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+      - name: Create PR
+        uses: peter-evans/create-pull-request@v4
+        with:
+          base: master
+          branch: docs/update-contributors
+          token: ${{ secrets.PAT }}
+          delete-branch: true
+          title: "chore: update contributors"
+          labels: Icons
```
Comment (DSil) [inline .github/workflows/cron-update-contributors.yml:38]: Do we want this label? Or is this a copy-paste leftover? 👀

File: docs/plugins/contributors/users.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "github": "https://github.com/CollierCZ",
     "twitter": "",
     "website": "https://collier.cz/",
+    "core": true,
```
Comment (DSil) [inline docs/plugins/contributors/users.json:13]: Is this property new? Why do we need it? 🤔

File: docs/src/components/Github/OrbitTeam.tsx [hunk 3]
```diff
@@ -8,16 +8,12 @@ import Member from "./TeamMember";
 
 import { Contributor } from ".";
 
-interface AllMembers {
+interface Props {
   current: Contributor[];
   previous: Contributor[];
```
Comment (DSil) [inline docs/src/components/Github/OrbitTeam.tsx:13]: I know these are old, but at least the interface would tell us it was about members. Now I think the readability would improve a lot if we could rename these to `currentMembers` and `previousMembers`, for example. WDYT?

File: .github/workflows/cron-update-contributors.yml (hunk 1 above)
Comment (mainframev) [reply .github/workflows/cron-update-contributors.yml:38]: good catch, not, for contributors we don't ;)

File: docs/plugins/contributors/users.json (hunk 2 above)
Comment (mainframev) [reply docs/plugins/contributors/users.json:13]: yeah, to distinguish core team from other contributors :)

File: docs/src/components/Github/OrbitTeam.tsx (hunk 3 above)
Comment (mainframev) [reply docs/src/components/Github/OrbitTeam.tsx:13]: sure, makes sense, will rename ;)

**Review by DSil — APPROVED** [review]
👍

**Pull request merged**